runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/CompositeParsers/DelegatorInvokesDelegateRuleWithArgs.stg

```diff
@@ -28,5 +28,5 @@ WS : (' '|'\n') -> skip ;
 
 slaveGrammar(grammarName) ::= <<
 parser grammar S;
-a[int x] returns [int y] : B {<write("\"S.a\"")> $y=1000;} ;
```

Comment:
Interesting. I think we could actually use the following as a more "ANTLR-y" way to do this. How do you feel about this as a general pattern for executing a sequence of multiple actions at a specific location in a test?

```
{<write("\"S.a\"")>} {$y=1000;}
```